cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -42,17 +33,10 @@ class MatchType(StrEnum):
     partial = "partial"
 
 
-class InformationMetadata(DomainMetadata):
+class InformationMetadata(BaseMetadata):
     role: ClassVar[RoleTypes] = RoleTypes.information_architect
-    prefix: Prefix = Field(
-        description="This is used as prefix for generation of RDF OWL/SHACL data model representation",
-    )
-    namespace: Namespace = Field(
-        description="This is used as RDF namespace for generation of RDF OWL/SHACL data model representation "
-        "and/or for generation of RDF graphs.",
-        min_length=1,
-        max_length=2048,
-    )
+    prefix: Prefix
+    namespace: Namespace_
```

Comment:
Same here, no collision with Python keyword or built-in
```suggestion
    namespace: Namespace
```

Reply:
It colides since Namespace is rdflib entity, while Namespace_ is type we define. but I have renamed all the types in _type with suffix Type